Automattic/wp-calypso
# Woo Express: Add plan feature comparison grid

Related to #75383

## Proposed Changes

This adds the "Compare plans" button at the bottom of the Woo Express `/plans` page. Clicking the button shows the expanded Woo Express feature comparison grid. The Plus plan is intentionally hidden from the comparison grid.

This is part of several PRs related to plan features and the plan comparison grid:
* #75918 (this PR) which adds all of the plan features and groups.
* #75769 which enables the plan comparison grid for Woo Express.
* #75883 which adds footnotes to the plan comparison grid.

### Collapsed `/plans` page
![Collapsed](https://user-images.githubusercontent.com/917632/232137172-d32357ef-0159-4bf0-8ba6-ece0a3e7910a.png)

### Expanded `/plans` page
![Expanded](https://user-images.githubusercontent.com/917632/232137211-c93aa805-9009-4528-8c68-8a450a493c64.png)

## Testing Instructions



* Apply this patch (or use the calypso.live link).
* Visit the `/plans` page for a Trial site or a site with the Essential plan.
* Click the "Compare plans" button and verify that the expanded feature list looks like the screenshot above.

This Slack conversation (p1681499986012049-slack-C0Q664T29) has a link to a spreadsheet with all of the features.
* Verify that the correct features are listed under the correct groups.

**NOTE: This doesn't include the footnotes shown in the Figma design. Footnotes are being added in #75883**

## Pre-merge Checklist
